src/config/shell.md

```diff
@@ -32,15 +32,18 @@ shells.
 
 - bash
 - dash
+- dreampie
```

Comment:
No

Reply:
This is not an available shell.

Reply:
My mistake. I Just went and installed all the shells I could find and looked at which ones added entries to /etc/shells. This one does not though.

Reply:
Removed it in the latest push